zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -1176,27 +1179,27 @@ impl<F: Field> ExecutionConfig<F> {
                         break;
                     }
                     second_last_real_step = Some(cur);
+                    // record offset of current step before assignment
+                    second_last_real_step_offset = offset;
```

Comment:
fix bug: before fixing, `second_last_real_step_offset` are recorded offset for last step, while what we needed is second last.